03-interactive-site/00-html-and-css/03-box-model-and-display/README.md

```diff
@@ -0,0 +1,372 @@
+# 03 - Modelo de Caixa e Display
```

Comment:
Let's avoid the enumeration in titles. Since the folder is enumerated, it will render in the same order in the LMS.